tests/views_integration/Tribe/Events/Views/V2/Partials/Components/MessagesTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Tribe\Events\Views\V2\Partials\Components;
+
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
+
+class MessagesTest extends HtmlPartialTestCase
+{
+
+	protected $partial_path = 'components/messages';
+
+	/**
+	 * Test render with context
+	 */
+	public function test_render_with_context() {
```

Comment:
let's test the loop by writing a test for multiple notices as well as multiple message groups.

Reply:
I like that idea.